language/stdlib/modules/LibraAccount.move

```diff
@@ -1127,7 +1263,7 @@ module LibraAccount {
     spec module {
         /// The LibraAccount under addr holds either no withdraw capability
         /// (withdraw cap has been delegated) or the withdraw capability for addr itself.
-        invariant [global, isolated] forall addr1: address where exists_at(addr1):
+        invariant [global] forall addr1: address where exists_at(addr1):
```

Comment:
Curious: did you need to remove isolated because some other proof depends on this?

Reply:
Yes in transaction scripts like this one where we extract the withdraw capability from the sending account and uses that withdraw capability, we need the property that the capability indeed belongs to the sending account.